run_visualizer_local.py

```diff
@@ -33,9 +30,9 @@
     while not os.path.exists(path_init):
         time.sleep(1)
         
-    opt = pickle.load(open(path_init,'rb'))
-    opt.remote = False
-    visualizer = Visualizer(opt)   # create a visualizer that display/save images and plots
+    params_opt = pickle.load(open(path_init,'rb'))
+    params_opt['remote'] = False
```

Comment:
@wendywwang why does this get set to False?

Reply:
You said you tested it, but once you confirm that this is expected I'll approve!

Reply:
@rich-nieto-ibm 
- when remote is false, you call Visualizer() and it will really start doing visdom stuff
- when remote is true, you call Visualizer() and it saves checkpoints and doesn't do anything about visdom

This opt snapshot we saved from training has to have a value of True for flag remote, otherwise you don't need a local visualizer launcher. Given that, to use the same code to call Visualizer, we need to manually turn off this flag, so that this time the visualizer class will really work on visdom.